cluster-up/cluster/kind-k8s-sriov-1.17.0/provider.sh

```diff
@@ -31,5 +33,35 @@ function up() {
 EOF
 
     setup_kind
+
+    # remove the rancher.io kind default storageClass
+    _kubectl delete sc standard
+
+    nodes=$(_kubectl get nodes -o=custom-columns=:.metadata.name | awk NF)
+    for node in $nodes; do
+        # Create local-volume directories, which, on other providers, are pre-provisioned.
+        # For more info, check https://github.com/kubevirt/kubevirtci/blob/master/cluster-provision/STORAGE.md
+        for i in {1..10}; do
+            mount_disk $node $i
+        done
+        $DOCKER exec $node bash -c "chmod -R 777 /var/local/kubevirt-storage/local-volume"
+
+        # Create a unique UUID file reference file for each node, that will be mounted in order to support
+        # Migration in Kind providers.
+        $DOCKER exec $node bash -c 'cat /proc/sys/kernel/random/uuid > /kind/product_uuid'
```

Comment:
please remove this as well, its was needed for the PodPreset

Reply:
Done